apps/sms/test/marionette/new_activity_test.js

```diff
@@ -102,7 +102,7 @@ marionette('Messages as "new" activity target', function() {
         return !!window.wrappedJSObject.Compose;
       }), 'XSS should not be performed');
 
-      assert.isTrue(messagesApp.Composer.sendButton.enabled());
+      client.waitFor(() => messagesApp.Composer.sendButton.enabled());
```

Comment:
Not quite sure about the timing of this `client.waitFor`, maybe we can wait for specific element in `launchAsActivity`?